.github/workflows/test.yaml

```diff
@@ -47,6 +47,21 @@ jobs:
             fi
           done
 
+      - name: Install plugin and run Unit-tests on changed charts
+        id: unit-tests
+        env:
+          CHARTS_CHANGED: ${{ steps.list-changed.outputs.charts_changed }}
+        run: |
+          helm plugin install https://github.com/quintush/helm-unittest
```

Comment:
let's make this step silent because it produces a lot of output

Reply:
Silenced this and also the dep update step inside the unit tests.